@here/harp-mapview/lib/geometry/overlayOnElevation.ts

```diff
@@ -20,6 +23,42 @@ function overlayObject(object: TileObject, displacementMap: THREE.DataTexture):
     }
 }
 
+function overlayTextPosition(
+    tileCoords: THREE.Vector3,
+    elevationProvider: ElevationProvider,
+    projection: Projection,
+    tile: Tile
+) {
+    // TODO: Calculate geoCoordinates in decoder.
```

Comment:
I think this comment should go where this function is used, this function by itself seems perfectly appropriate.

Reply:
done